go/scanner/scanner.go

```diff
@@ -150,7 +150,7 @@ func DefaultScannerOptions() *ScannerOptions {
 // Scanner is a tool to scan all the entries in a CT Log.
 type Scanner struct {
 	// Client used to talk to the CT log instance
-	logClient *client.LogClient
+	logClient client.LogClient
```

Comment:
There was nothing wrong with the original code, but the consequence of the interface extraction is that we now pass interfaces around, and this is not done using pointers in Go from what I've read.